content/en/quality_gates/explorer/search_syntax.md

```diff
@@ -0,0 +1,87 @@
+|--------------|--------------------------------------------------------------------------------------------------------|------------------------------|
+| `AND`        | **Intersection**: both terms are in the selected events. `AND` is the default if no operator is specified between terms.) | `authentication AND failure`   |
+| `OR`         | **Union**: either term is contained in the selected events                                             | `authentication OR password`   |
+| `-`          | **Exclusion**: the following term is NOT in the event (apply to each individual raw text search)                                                  | `authentication AND -password` |
+
+## Search on attributes and tags
+
+You are not required to define a facet in order to search on attributes and tags. To search on a specific attribute, add `@` to specify you are searching on an attribute. Attributes searches are case sensitive. Use free text search to get case insensitive results. 
+
+For example, if you are interested in the `git.repository.name` attribute and you want to filter on the value `Datadog/documentation`, use `@git.repository.name:DataDog/documentation`.
+
+To search for an attribute value that contains special characters, escape it with a backslash or use double quotes. For example, for an attribute `my_attribute` with the value `hello:world`, search using: `@my_attribute:hello\:world` or `@my_attribute:"hello:world"`.
+
+To match a single special character or space, use the `?` wildcard. For example, for an attribute `my_attribute` with the values `hello world`, `hello-world`, or `hello_world`, search using: `@my_attribute:hello?world`.
+
+For more information about tags, see [Using Tags][2].
+
+## Wildcards
+
+### Multi-character wildcard
+
+To perform a multi-character wildcard search, use the `*` symbol as follows:
+
+* `service:web*` matches every log message that has a service starting with `web`.
+* `web*` matches all log messages starting with `web`.
+* `*web` matches all log messages that end with `web`.
+
+Wildcard searches work within tags and attributes (faceted or not) with this syntax. This query returns all the services that end with the string `mongo`:
+
+```
+test.service:*mongo
+```
+
+### Search wildcard
+
+When searching for an attribute or tag value that contains special characters or requires escaping or double quotes, use the `?` wildcard to match a single special character or space. For example, to search for an attribute `my_attribute` with the value `hello world`, `hello-world`, or `hello_world`, use `@my_attribute:hello?world`.
+<p> </p>
+
+## Numerical values
+
+To search for a numerical attribute, first [add it as a facet][1]. You can then use numerical operators (`<`,`>`, `<=`, or `>=`) to perform a search on numerical facets.
+
+For example, to retrieve all test runs that have a duration of over one week, use: `@duration:>=7days`.
```

Comment:
I think we can get rid of Numerical values section, since we don't have any numerical values in our filter facets for quality gates